apache/superset
# feat: Add loadBalancerSourceRanges to helm configuration

### SUMMARY

Adds `loadBalancerSourceRanges` to helm configuration in service block to allow for easy whitelisting of IPs.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TEST PLAN

1. Deploy as a LoadBalancer with `loadBalancerSourceRanges: [ YOUR-IP ]` and confirm that you can reach the service
2. Deploy as a LoadBalancer with `loadBalancerSourceRanges: [ ]` and confirm that you cannot reach the service

### ADDITIONAL INFORMATION
